Library/Homebrew/cmd/search.rb

```diff
@@ -53,7 +54,7 @@ def search_args
       package_manager_switches = PACKAGE_MANAGERS.keys.map { |name| "--#{name}" }
       package_manager_switches.each do |s|
         switch s,
-               description: "Search for <text> in the given package manager's list."
+               description: "Search for <text> in the given database's or package manager's list."
```

Comment:
```suggestion
               description: "Search for <text> in the given list."
```

maybe?

Reply:
Oops, I slept on this one and forgot I had auto-merge activated. How about

```
Search for <text> in the given database.
```
?

Reply:
#11748